pkg/apis/v1beta1/nodepool_validation_cel_test.go

```diff
@@ -632,5 +632,30 @@ var _ = Describe("CEL/Validation", func() {
 				nodePool = oldNodePool.DeepCopy()
 			}
 		})
+		It("should allow subdomain labels in restricted domains exceptions list", func() {
```

Comment:
Is it worth adding this same validation for the NodeClaims validations as well?

Reply:
added